packages/vkui/src/components/CarouselBase/types.ts

```diff
@@ -27,6 +20,65 @@ export interface LayoutState {
   isFullyVisible: boolean;
 }
 
+export interface LoopPoint {
+  /**
+   * Индекс слайда
+   */
+  index: number;
+  /**
+   * Функция, которая по текущему сдвигу галереи определяет нужный сдвиг слайда
+   */
+  target: (this: void, location: number) => void;
+}
+
+export interface ControlElementsState {
+  /**
+   * Отвечает за отображение стрелки влево
+   */
+  canSlideLeft: boolean;
+  /**
+   * Отвечает за отображение стрелки вправо
+   */
+  canSlideRight: boolean;
+  /**
+   * Возможность листаться слайды drag'ом
```

Comment:
```suggestion
   * Возможность листать слайды drag'ом
```
ой, была опечатка изначально